### PR TITLE
added mcc

### PR DESCRIPTION
**Pull request by behrica**

### CODE REVIEW

**Comment by shark8me** [comment]
looks good to me.

**Comment by shark8me** [comment]
will see if I can do a new release with MCC included

**Comment by behrica** [comment]
Maybe i should add the multi label version. I have the formula from sklearn.
